src/list.scss

```diff
@@ -164,6 +174,36 @@ $block: #{$fd-namespace}-list;
     }
   }
 
+  &__link {
+    @include fd-reset();
+
+    display: flex;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+    padding: 0 $fd-list-horizontal-padding;
+    cursor: pointer;
+
+    @include fake-list-outline();
```

Comment:
We need an offset for the outline and the offset property is not supported in IE11. This solution was discussed few weeks ago and we agreed to use it (if possible, because it relies on ::before). Any other solutions are more than welcome.